apps/web/app/quantity-leaf-slider.tsx

```diff
@@ -1,12 +1,35 @@
+import { useMemo } from 'react'
+
```

Comment:
name does not match the file name. And name is not very clear.

Idea
Slider

?